experimental/packages/opentelemetry-instrumentation/src/platform/node/instrumentationNodeModuleFile.ts

```diff
@@ -18,7 +18,7 @@ import { InstrumentationModuleFile } from './types';
 import { normalize } from 'path';
 
 export class InstrumentationNodeModuleFile<T>
-  implements InstrumentationModuleFile<T> {
+implements InstrumentationModuleFile<T> {
```

Comment:
Previously I've tried to apply the indent rule but with no luck, the typescript ast nodes are not indented properly: https://github.com/typescript-eslint/typescript-eslint/issues/1824.

Or maybe it is fine to ignore this indentation problem?

Reply:
Right, but not sure if there is much we can do currently 🤷‍♂️
We can either ignore indentation linting altogether or live with the typescript issue until it is resolved.
WDYT?

Reply:
I'm fine with it.